src/ocaml_version.mli

```diff
@@ -24,3 +24,13 @@ val supports_response_file : t -> bool
 
 (** Does ocamlmklib support [-args0]? *)
 val ocamlmklib_supports_response_file : t -> bool
+
+(** Whether [Pervasives] includes the [result] type *)
+val pervasives_includes_result : t -> bool
+
+(** Whether the standard library includes the [Uchar] module *)
+val stdlib_includes_uchar : t -> bool
+
+(**/**)
+
+val for_test_only : t
```

Comment:
Actually, why not add a function `make : int * int * int -> t`?

Reply:
Good idea; done.